### PR TITLE
Add NAPPTIVE Playground CLI feature to the collection index

### PR DESCRIPTION
**Pull request by oleksis**

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -153,3 +153,8 @@
   contact: https://github.com/jsburckhardt/devcontainer-features/issues
   repository: https://github.com/jsburckhardt/devcontainer-features
   ociReference: ghcr.io/jsburckhardt/devcontainer-features
+- name: NAPPTIVE Playground CLI Features
+  maintainer: oleksis
+  contact: https://github.com/oleksis/playground-cli/issues
+  repository: https://github.com/oleksis/playground-cli
+  ociReference: ghcr.io/oleksis/playground-cli/playground-cli
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/oleksis/playground-cli
```

The `ociReference` is the `${registry}/${owner}/${repoName}` without the `id` part.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks for the contribution. ✨ 
Left a comment.

**Comment by oleksis** [comment]
Thanks for the feedback. Suggestions applied!

**Review by samruddhikhandale — APPROVED** [review]
🚀

**Pull request merged**